metadata.json

```diff
@@ -7,6 +7,7 @@
   "source": "https://github.com/simp/pupmod-simp-sssd",
   "project_page": "https://github.com/simp/pupmod-simp-sssd",
   "issues_url": "https://simp-project.atlassian.net",
+  "data_provider": "hiera",
```

Comment:
README needs to be updated to let people know that they should go look for the truth of the parameter values in the `hiera` stack in the module. Also, need to tell people how to use `puppet lookup` so they have some idea on how to troubleshoot this module.